### ISSUE TITLE
GetJavaRDDOfAllElements and GetRDDOfAllElements return edges twice

### ISSUE THREAD

**Issue by gaffer01** [gchq/Gaffer#1245]
The Accumulo store handlers for `GetJavaRDDOfAllElements` and `GetRDDOfAllElements` return each edge twice. The behaviour should be consistent with `GetAllElements` and only return each edge once.

It would make sense to fix this after #1222 has been merged in as that contains some significant improvements to the unit tests for the handlers for these operations.

**Comment by p013570** [comment]
Merged into develop.
